cisco_secure_endpoint/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+[Cisco Secure Endpoint][1] is a single-agent solution that provides comprehensive protection, detection, response, and user access coverage to defend against threats to your endpoints. Using cutting-edge technology, it detects and neutralizes malicious activity in real-time, ensuring robust protection for your digital assets.
+
+This integration ingests the following logs:
+- Audit: Audit logs provide activities performed by user in Cisco Secure Endpoint console.
+- Event: Event logs are essential for tracking security events, enabling quick detection, response, and analysis of potential threats.
+
+The Cisco Secure Endpoint integration provides OOTB Dashboard visualizations to gain quick insights into the Cisco Secure Endpoint's audit  and event logs, thus enabling quick and necessary actions. Also, OOTB detection rules are designed to help users monitor and respond to potential security threats effectively.
+
+
+## Setup
+
+### Configuration
+
+#### Get API Credentials for Cisco Secure Endpoint 
+
+
+--> Refer the below Steps to create Client ID and API Key:
+1. Log in to your Cisco Secure Endpoint Console. Click on the Left side Menu Panel.
+2. Select `Administration`, Inside that select `Organization Settings`.
+3. Click `Configure API Credentials` under `Features` section,  to generate the new API Credentials.
+4. Click on the `New API Credentials` button located at the right side under section `Legacy API Credentials (version 0 and 1)`.
+5. Add the below details in the pop-up:
+    - Application Name: Any preferable name
+    - Scope: Select `Read-only`
+    - Click on `Create`.
+    - Once you click on create, the redirected page will display the client ID(i.e: 3rd Party API client ID) and API Key values.
+        - NOTE: Please make a note of the API Key, as it will only be displayed once.
```

Comment:
```suggestion
    - Application Name: Any preferable name.
    - Scope: Select `Read-only`.
    - Click `Create`.
    - Once you click **Create**, the redirected page will display the client ID (like a third party API client ID) and API Key values.
        - **Note:** Make a note of the API Key, as it will only be displayed once.
```

Reply:
Thanks for the comment. Have made the changes.